react-components/src/library/components/assign-to-class/assign-modal.js

```diff
@@ -221,12 +220,21 @@ export default class AssignModal extends React.Component {
     )
   }
 
+  openConfirmModal () {
+    this.setState({ showModal: true })
+  }
```

Comment:
This function doesn't seem to be used, since the trigger for opening the modal is the success of the AJAX request.

Reply:
Ah, right. Thanks for catching that.